### PR TITLE
Change default ads per hour via variations service - 1.21.x

### PR DESCRIPTION
**Pull request by tmancey**
Uplift of https://github.com/brave/brave-core/pull/8086 & https://github.com/brave/brave-core/pull/8206
Resolves https://github.com/brave/brave-browser/issues/14377 & https://github.com/brave/brave-browser/issues/14605

- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR.
- [x] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by kjozwiak** [comment]
Waiting on https://github.com/brave/brave-core/pull/8206.

**Comment by tmancey** [comment]
https://github.com/brave/brave-core/pull/8206 has been merged

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.21.x` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-core/pull/8086#issuecomment-799052682.

**Pull request merged**
